### DISCUSSION TITLE
Backup results in "Error initialising app: CouchDB error: missing"

### DISCUSSION THREAD

**Discussion by stellarfinn** [Budibase/budibase#14631] [Help] [answered]
Title says it all. I restored a back up and this is the screen I'm getting now. 

![image](https://github.com/user-attachments/assets/312c7344-e057-45ef-8434-978e680972cf)

**Comment by stellarfinn** [comment]
@melohagan @joebudi Sorry to tag! I'm in big trouble here! =(

**Reply by stellarfinn** [reply]
@shogunpurple

**Comment by stellarfinn** [comment]
Also, I can view this screen and the app is still functional but I can't edit the app. 

<img width="1913" alt="image" src="https://github.com/user-attachments/assets/d25f5748-2a79-4b48-b439-960f3139bb7a">

**Reply by stellarfinn** [reply]
@aptkingston Any thoughts?

**Comment by stellarfinn** [comment]
It's gotten really odd. I've had this screen for the past couple hours. I finally said screw it and just filled it out with "test". 

<img width="1773" alt="image" src="https://github.com/user-attachments/assets/9c5c6008-f255-4ce9-958d-28d1deb2687e">

An now my app is completely gone, BUT it's still functional on the web side... 

https://ciaoimports.budibase.app/app/portal#/brands/pricelist/wholesale/ro_ta_b3408e3a19c64d58bb547ef8dc15a31e_35565376ccdf468fbbf3ec6af164d956/Ajinomoto/Fixed/Wholesale/In%20Stock/

this link is still operational, a public link, and I still have my app open in another tab which is still functional. Very odd.



<img width="1914" alt="image" src="https://github.com/user-attachments/assets/5d6200cc-5758-471c-b82c-7cbe5d113c38">

**Reply by melohagan** [reply]
Hey @stellarfinn Sorry that you're having issues. Our support team is looking at it now.

**Comment by stellarfinn** [comment, marked as answer]
Fixed by the Budibase team.

**Reply by Kry9toN** [reply]
i with same issue how to fix it?

**Reply by ConorWebb96** [reply]
Hey @Kry9toN,

Can you send an email into `support@budibase.com`.

Can you also provide the following information;
- Tenant ID
- App id
- Screenshot of the error message
- Network and console log screenshots are also helpful.

**Comment by turnerzzzz** [comment]
I’m experiencing the same issue and I’m panicking. app is now gone too. I apologize for tagging you.
I’ve already sent the details to support@budibase.com.
ieducationm_8da67d5c9de54e949d1ba5cbc48f52d5
tenant: ieducationm.budibase.app

@ConorWebb96 @melohagan @mikesealey

**Reply by Kry9toN** [reply]
My case is already fixed, but the fix happens gradually, so I keep waiting; eventually, the database will correct itself. That's why I don't mention it here, out of concern that it might not happen for you.
